romancal/scripts/static_preview.py

```diff
@@ -1,14 +1,13 @@
 from pathlib import Path
-from typing import TYPE_CHECKING, Optional
+from typing import Optional
 
 import asdf
 import numpy
 
-if TYPE_CHECKING:
-    from typing_extensions import Annotated
-
 
 def command():
+    from typing_extensions import Annotated
```

Comment:
If this is supposed to be available at runtime then you must include this as one of the listed dependencies.

Reply:
Its probably fine, especially in a try block. I'm just thinking about what would happen if typer dropped the dependency, which it probably will do eventually.